utils/src/ooinstall/cli_installer.py

```diff
@@ -261,23 +307,32 @@ def check_hosts_config(oo_cfg):
             if master_lb[0].master or master_lb[0].node:
                 click.echo('The Master load balancer is configured as a master or node. Please correct this.')
                 sys.exit(0)
+            # Check for another host with same connect_to?
         else:
             message = """
-No HAProxy given in config. Either specify one or provide a load balancing solution
-of your choice to balance the master API (port 8443) on all master hosts.
+WARNING: No master load balancer specified in config. If you proceed you will
+need to provide a load balancing solution of your choice to balance the
+API (port 8443) on all master hosts.
```

Comment:
Personally I'd prefer the fewest prompts possible.  For internal traffic I would think they could use the openshift or kubernetes Services exposed by the kube-proxy in most cases.  @detiber might know of a reason why someone might want an additional internal load balancer.